src/bam/collapse_reads_to_fragments/calc_consensus.rs

```diff
@@ -93,26 +91,36 @@ impl<'a> CalcOverlappingConsensus<'a> {
             };
             self.build_consensus_strand(&mut consensus_strand, consensus_seq[i], i);
         }
+        let mut tag_string = Vec::new();
+        match self.recs1()[0].read_pair_orientation().as_ref() {
+            "R1R2" | "R2R1" => {
+                consensus_seq = revcomp(consensus_seq);
+                consensus_qual.reverse();
+            }
+            "R1F2" | "F1R2" | "R2F1" | "F2R1" => {
+                tag_string = [tag_string, b"SI:Z:".to_vec(), consensus_strand, vec![b' ']].concat();
```

Comment:
I don't like this kind of concatenation for `Vec<u8>` as I need to at whitespaces manually. Is there something equivalent for joining Vectors with a separator as for Strings?

Reply:
Appears to exist for slices? https://doc.rust-lang.org/std/vec/struct.Vec.html#method.join